default_data.json

```diff
@@ -34048,11 +34048,15 @@
             "companies": [
                 {
                     "company_name": "99cloud",
+                    "end_date": "2020-Jun-01"
+                },
+                {
+                    "company_name": "cetccloud",
```

Comment:
Please add company to the company list. Like here https://github.com/stackalytics/default_data/pull/187/files